jest-common/src/main/java/io/searchbox/indices/script/DeleteIndexedScript.java

```diff
@@ -0,0 +1,64 @@
+package io.searchbox.indices.script;
+
+import io.searchbox.action.AbstractAction;
+import io.searchbox.action.GenericResultAbstractAction;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
+import static io.searchbox.indices.script.ScriptLanguage.GROOVY;
+
+public class DeleteIndexedScript extends GenericResultAbstractAction {
+
+  private final String scriptName;
+  private final ScriptLanguage scriptLanguage;
+
+  public DeleteIndexedScript(Builder builder) {
+    super(builder);
+    this.scriptName = builder.scriptName;
+    this.scriptLanguage = builder.scriptLanguage;
+    setURI(buildURI());
+  }
+
+  protected String buildURI() {
+    try {
+      return super.buildURI() + "/_scripts/" + scriptLanguage.pathParameterName + "/" + URLEncoder.encode(scriptName, CHARSET);
+    } catch (UnsupportedEncodingException e) {
+      throw new RuntimeException(e);
```

Comment:
Different behaviour than `super.buildURI` (and well actually than all places where `UnsupportedEncodingException` was thrown)... Please keep it consistent - either change this or all other ones. I'm not sure which way to go as this only expected if the class is extended and charset is overridden.